### DISCUSSION TITLE
allowing duplicate email in modules

### DISCUSSION THREAD

**Discussion by randysrs** [YetiForceCompany/YetiForceCRM#15920] [How to]
so i created a email field in a new module specific to this module and in some cases its the same email as the primary account email .
this then triggers Yeti error "This email address already exists in this **module.**  , but not sure if its looking at the original account module or the new module we created .

**Comment by rskrzypczak** [comment]
If you created a new module, then there is no duplicate validation.
Duplicate verification is set only in the Accounts module and the Contacts module. Verification works only within one module, i.e. if you create/edit a Account's entry and receive such a message, it means that there is another entry in the module that already has this email entered in some e-mail field (the entry may even be in the trash bin).
![image](https://user-images.githubusercontent.com/10221999/162883887-464495ab-a5a0-4363-81b6-5ef9807c577b.png)
